django/core/files/move.py

```diff
@@ -66,7 +67,15 @@ def file_move_safe(old_file_name, new_file_name, chunk_size=1024 * 64, allow_ove
         finally:
             locks.unlock(fd)
             os.close(fd)
-    copystat(old_file_name, new_file_name)
+
+    try:
+        copystat(old_file_name, new_file_name)
+    except PermissionError as e:
+        # Certain filesystems (CIFS) fail to copy the files metadata if
+        # the destination filesystem is not the same type of filesystem
+        # as the source, ignore it.
+        if getattr(e, 'errno', 0) != errno.EPERM:
```

Comment:
I think it would be safe to access `e.errno` directly in this case. The `winerr` attribute is only present on Windows platform hence the `getattr()`.